boards/riscv/it8xxx2_evb/it8xxx2_evb.dts

```diff
@@ -12,11 +12,6 @@
 	model = "IT8XXX2 EV-Board";
 	compatible = "riscv,it8xxx2-evb";
 
-	aliases {
```

Comment:
Move the changes to the EVB board into a separate commit.

Reply:
Done